web/src/services/config/index.ts

```diff
@@ -80,9 +81,9 @@ export default async function loadConfig() {
     config.extensions = extensions;
   }
 
-  config.unsafeBuiltinPlugins = await loadUnsafeBuiltinPlugins();
-
   window.REEARTH_CONFIG = config;
+
+  config.unsafeBuiltinPlugins = await loadUnsafeBuiltinPlugins();
```

Comment:
Why are you loading unsafeBuiltinPlugins after we assign config to window.REEARTH_CONFIG?